dev/scripts/src/alluxio.org/build-distribution/cmd/generate-tarball.go

```diff
@@ -334,7 +335,7 @@ func generateTarball(hadoopClients []string) error {
 	addAdditionalFiles(srcPath, dstPath, hadoopVersion, version)
 
 	chdir(cwd)
-	run("creating the distribution tarball", "tar", "-czvf", tarball, dstDir)
+	run("creating the distribution tarball", "tar", "-czpvf", tarball, dstDir)
```

Comment:
Does this `-p` do anything? My understanding is that permissions are preserved by default on the create path, and `-p` only affects the extract path

Reply:
left in from me figuring out the same thing :). Fixed.